eng/docker-compose/local-dms.yml

```diff
@@ -5,7 +5,10 @@
 
 services:
   dms:
-    build: ../../src/dms/
+    build:
+      context: ../../src/dms/
+      additional_contexts:
+        parentdir: ../../src/
```

Comment:
This allows us to copy files from a parent directory in addition to the primary context directory.